src/components/controllers/repl/index.jsx

```diff
@@ -52,6 +54,13 @@ export function Repl({ code }) {
 		}
 	};
 
+	useEffect(() => {
+		const delay = setTimeout(() => {
+			setRunnerCode(editorCode);
+		}, 250);
```

Comment:
Happy to revert, but 250ms felt a bit better? This only affects the REPL runner pane, not the editor itself, so it's not displaying errors that get in the user's way.

Just a bit more responsive for simple edits is all.